File: app/src/main/java/org/mnmv/sirius/General.java
package org.mnmv.sirius;

import android.content.Intent;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuInflater;
import android.view.MenuItem;
import android.widget.Toast;

import com.google.android.material.bottomnavigation.BottomNavigationView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

public class General extends AppCompatActivity {

    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.u_general);

        //Верхняя
        Toolbar myToolbar = findViewById(R.id.sirius_bar);
        setSupportActionBar(myToolbar);

        //Нижняя Панелька
        BottomNavigationView bottomNavigationView = findViewById(R.id.bnve);
        Menu menu = bottomNavigationView.getMenu();
        MenuItem menuItem = menu.getItem(1);
        menuItem.setChecked(true);
        bottomNavigationView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
            @Override
            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                switch (item.getItemId()) {
                    case R.id.nav_marks:
                        Intent int_to_posts = new Intent(General.this, Marks.class);
                        startActivity(int_to_posts);
                        overridePendingTransition(0, 0);
                        finish();
                        break;
                    case R.id.nav_menu:

                        break;
                    case R.id.nav_sets:
                        Intent int_to_sets = new Intent(General.this, Settings.class);
                        startActivity(int_to_sets);
                        break;
                }
                return false;
            }
        });
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        MenuInflater menuInflater = getMenuInflater();
        menuInflater.inflate(R.menu.nav_menu, menu);
        return true;
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()) {
            case R.id.nav_sets:
                Toast.makeText(this, "Скоро", Toast.LENGTH_SHORT).show();
                break;
            default:
                return super.onOptionsItemSelected(item);
        }
        return super.onOptionsItemSelected(item);
    }
}

File: app/src/main/java/org/mnmv/sirius/General.kt
package org.mnmv.sirius

import android.app.ActionBar
import android.content.Intent
import android.os.Bundle
import android.view.Menu
import android.view.MenuItem
import android.view.animation.AnimationUtils
import android.widget.FrameLayout
import android.widget.ImageView
import android.widget.Toast

import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.bottomappbar.BottomAppBar
import com.google.android.material.bottomappbar.BottomAppBar.FAB_ALIGNMENT_MODE_CENTER
import com.google.android.material.bottomappbar.BottomAppBar.FAB_ALIGNMENT_MODE_END
import com.google.android.material.floatingactionbutton.FloatingActionButton

class General : AppCompatActivity() {

    private var currentFabAlignmentMode = FAB_ALIGNMENT_MODE_CENTER
    private var currentFabAlignmentMode2 = FAB_ALIGNMENT_MODE_END

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.general)

        //Верхняя
        val bar = findViewById<BottomAppBar>(R.id.bottom_app_bar)
        setSupportActionBar(bar)
        bar.fabAlignmentMode = currentFabAlignmentMode

        val fab_toMarks = findViewById<FloatingActionButton>(R.id.fab_marks)
        fab_toMarks.setOnClickListener {
            val go_toMarks = Intent(this@General, MM::class.java)
            startActivity(go_toMarks)
            overridePendingTransition(0, 0)
            finish()
        }
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        val menuInflater = menuInflater
        menuInflater.inflate(R.menu.nav_menu, menu)
        return true

    }



    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            R.id.nav_sets -> {
                val go_to_sets = Intent(this, Settings::class.java)
                startActivity(go_to_sets)
            }
            android.R.id.home -> {
                val bndf = BottomNavigationDrawerFragment()
                bndf.show(supportFragmentManager, "dialog")
            }
            else -> return super.onOptionsItemSelected(item)
        }
        return super.onOptionsItemSelected(item)

    }

}